app-sso/tutorials/app-operators/claim-credentials.hbs.md

```diff
@@ -0,0 +1,109 @@
+# Claim credentials for an AppSSO service offering
+
+In the previous section we have seen how to discover the available AppSSO
```

Comment:
**[issue]**: referencing previous section. If tutorials are to be self-contained, then there shouldn't be navigational context. If they are sequential, then I would link to the previous section to ensure that if someone does land on this page without context, they will know to first read the _previous section_

Reply:
[ack](https://github.com/mamachanko/docs-tap/pull/1#discussion_r1223950777)